bin/crew

```diff
@@ -1273,7 +1273,7 @@ def install_files(prefix, path, dst = File.join( CREW_PREFIX, src.delete_prefix(
   return false unless Dir.exist?(src)
 
   if File.executable?("#{CREW_PREFIX}/bin/crew-mvdir") && !CREW_DISABLE_MVDIR
-    system 'crew-mvdir', @short_verbose, src, dst, exception: true
+    system 'crew-mvdir', *(@verbose ? [@short_verbose, src] : src), dst, exception: true
```

Comment:
Why the `*`?

Reply:
I think `*` is meant to collect the arguments as a list.

Reply:
In any case it doesn't work...

Reply:
This works: ` system "crew-mvdir #{@short_verbose} #{src} #{dst}",exception: true`

But worth noting that @verbose and @short_verbose are undefined in this function no matter whether `-v` is passed to install or not...

Reply:
I fixed the `@verbose` issue.